Cargo.toml

```diff
@@ -27,6 +27,7 @@ serde={version="1.0", features=["derive"], optional = true}
 num-traits = {version = "0.2", optional = true}
 itertools = {version = "0.11.0", optional = true}
 ipnet = {version="2.7", optional = true}
+bitflags = {version="2.3.3", features = ["serde"], optional = true}
```

Comment:
`enum-primitive-derive` is no-longer needed, right?

Reply:
I am thinking it may be better to replace it with the [`num_enum`](https://crates.io/crates/num_enum) crate. It does all of the same things as `enum-primitive-derive`, but is feature rich and appears to have more support. It supports deriving a catch-all variant which would remove the need to manually implement `From` the way I did for `AttrType`.

Reply:
Sounds good! Let's replace it with num_enum